docs/permissions/custom-rules.md

```diff
@@ -135,25 +139,49 @@ class TestPermissionPolicy implements PermissionPolicy {
 
 Now that we have a custom rule defined and added to our policy, we need provide it to the catalog plugin. This step is important because the catalog plugin will use the rule's `toQuery` and `apply` methods while evaluating conditional authorize results. There's no guarantee that the catalog and permission backends are running on the same server, so we must explicitly link the rule to ensure that it's available at runtime.
 
-The api for providing custom rules may differ between plugins, but there should typically be some integration point during the creation of the backend router. For the catalog, this integration point is exposed via `CatalogBuilder.addPermissionRules`.
-
-```typescript title="packages/backend/src/plugins/catalog.ts"
-import { CatalogBuilder } from '@backstage/plugin-catalog-backend';
-/* highlight-add-next-line */
-import { isInSystemRule } from './permission';
-
-...
-
-export default async function createPlugin(
-  env: PluginEnvironment,
-): Promise<Router> {
-  const builder = await CatalogBuilder.create(env);
-  /* highlight-add-next-line */
-  builder.addPermissionRules(isInSystemRule);
-  ...
-  return router;
-}
-```
+The api for providing custom rules may differ between plugins, but there should typically be some extension point that you can use to create a module that adds your rule. For the catalog, this extension point is exposed via `catalogPermissionExtensionPoint`. Here's the steps you'll need to take to add the `isInSystemRule` we created above to the catalog:
```

Comment:
Could be nice here to link to https://backstage.io/docs/backend-system/architecture/extension-points/ 

Maybe a NIT? `but there should typically be an extension point that you can [use/consume?] in your created module to add your rule`

Its also fine the way it is, maybe it just sounds better in my head 😅

Reply:
Added a link and adjusted to match what you suggested. Can you give it another look to make sure I got it right, please?